code/modules/projectiles/guns/revolvers.dm

```diff
@@ -531,8 +531,8 @@
 	)
 	attachable_offset = list("muzzle_x" = 29, "muzzle_y" = 22,"rail_x" = 11, "rail_y" = 25, "under_x" = 20, "under_y" = 18, "stock_x" = 20, "stock_y" = 18)
 
-	fire_delay = 1.2 SECONDS
+	fire_delay = 0.15 SECONDS
```

Comment:
```suggestion
	fire_delay = 0.25 SECONDS
```

Reply:
why should the cmb have more fire delay than a tp-44 anyway
i get i gave it 3 round burst memes with low fire delay but that's still less overall damage in the amount of time a tp-44 snubnose could output damage
It'd probably be better with 2 round burst and 0.2 though imo, and remove the damage mult that makes it do 1.5 more points of damage